nautobot_circuit_maintenance/views.py

```diff
@@ -204,7 +204,11 @@ def get_extra_context(self, request, instance):
             parsed_notification = models.ParsedNotification.objects.filter(raw_notification=instance).last()
         else:
             parsed_notification = None
-        return {"parsed_notification": parsed_notification}
+        raw_bytes = instance.raw.tobytes()
+        # TODO: Using unicode-escape to handle bytes converted from string from migration 0007
+        # This can be removed when squashing migrations on release 1.0.0
+        raw_repr = raw_bytes.decode("unicode-escape")
```

Comment:
This also seems wrong to me - shouldn't `instance.raw` already be `bytes`? Why do we need to call `tobytes` on it? I also don't like seeing "temporary" code here - it's not clear to me on the comment why this is needed now but won't be later?

Reply:
instance.raw is a `memoryview` that I'm converting to `bytes` above.
the "temporary" code is because for the converted notifications when migration from Text to Binary, the escaped characters are converted from `\n` to `\\n`. If we found a better way to do the conversion we could skip this "unicode-escape" and leave as `decode()`